.github/CODEOWNERS

```diff
@@ -13,7 +13,7 @@ environment*.yml @sufikaur
 
 /docs/ @MarcusHolly @adam-a-a
 
-/tutorials/ @adam-a-a @zacharybinger @avdudchenko @MarcusHolly @TimBartholomew
+/tutorials/ @adam-a-a @avdudchenko @MarcusHolly
```

Comment:
If we want to get everything done in one PR, we should add @ kurbansitterley to the tutorial list and close #1631. Or we could just merge the PRs separately. Either works...

Reply:
@MarcusHolly Added him